package.json

```diff
@@ -27,10 +27,10 @@
   },
   "dependencies": {
     "adm-zip": "^0.4.13",
-    "extract-zip": "1.6.6",
+    "extract-zip": "1.7.0",
```

Comment:
Could we change these dependencies to something like `^1.7.0` allowing consumers to upgrade minor patches without requiring an upgrade from this package?